api/http/api_deployments.go

```diff
@@ -1093,15 +1093,24 @@ func (d *DeploymentsApiHandlers) DeploymentsPerTenantHandler(w rest.ResponseWrit
 	tenantID := r.PathParam("tenant")
```

Comment:
This entire handler could simply be:
```
func (d *DeploymentsApiHandlers) DeploymentsPerTenantHandler(w rest.ResponseWriter, r *rest.Request) {
	r.Request = r.WithContext(identity.WithContext(
		r.Context(),
		&identity.Identity{Tenant: r.PathParam("tenant")},
	))
	d.LookupDeployment(w, r)
}
```

Reply:
https://github.com/mendersoftware/deployments/pull/514/commits/8299bed94ead03d7d22186ec0b3907119b5d9516